lib/engine/step/g_18_cz/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
```

Comment:
this needs to require BuySellParShares

Reply:
Then I did not understand the last commment correctly. 
This is needed because every coroporations has different par values.